mautic/mautic
# [UX] Improve the explanation of the Automatic way to implement the form

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [x]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:
This update introduces a clearer explanation for the embedded form implementation method. Previously, users lacked the information that any modifications made to the form within the platform would synchronize across all embedded instances. 

This enhancement ensures users are fully aware of the dynamic editing capabilities, leading to a more intuitive and efficient content management experience. This is particularly beneficial for users who manage forms across multiple pages, streamlining their workflow and reducing the need for repetitive updates.



#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Open Forms
3. Create a form
4. Save & close the form
5. Click on Automatic 
6. Read the description
![image](https://github.com/mautic/mautic/assets/116097999/fb584b3e-23b6-4080-92bb-c5c3fed7c57f)
